index.html

```diff
@@ -8,27 +8,30 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <nav id="navbar">
-      <div>
-        <a href="#projects">Ir hacia project</a>
```

Comment:
Great work so far, You have come a long way, keep moving forward👏 👍
> 
[ ] I kindly suggest you to change your `<a> </a>` for a `<h1> </h1>`, because the welcome is an important part of your code, and making this change gonna be more productive for SEO searching.